src/client/components/AddResume/AddResume.js

```diff
@@ -0,0 +1,45 @@
+import React, {useState} from 'react';
+import ReactDom from 'react-dom';
+import Button from '../Button/Button';
+import './AddResume.css';
+import close from '../../assets/images/closeIcon.svg';
+
+export const AddResume = () => {
+const[title,setTitle] = useState('');
+const[description,setDescription] = useState('');
+const[uploadFile,setUploadaFile] =useState('');
+
+    return ReactDom.createPortal ((
+        <div className = "popup">
```

Comment:
You have lots of spaces, please fix that

```suggestion
        <div className="popup">
```